shopinvader/services/address.py

```diff
@@ -148,6 +153,12 @@ def _validator_create(self):
             "opt_in": {"coerce": to_bool, "type": "boolean"},
             "opt_out": {"coerce": to_bool, "type": "boolean"},
             "lang": {"type": "string", "required": False},
+            "one": {
```

Comment:
@JordiMForgeFlow It would be nice to add some documentation on this new paramerter by using the 'meta' key.
```suggestion
            "one": {
                "meta": {
                    "description": "If true, the response of the create/update should return the single created record. If not passed, or passed as false, then return the list of customer addresses "
                }
```

`one` is not really self explanatory but I don't find a better naming.

Reply:
Thanks for the suggestion @lmignon, I didn't know about the meta key.

On the naming of the parameter, I originally named it 'response_one', but changed it to 'one' to align with the similar approach that @simahawk took in https://github.com/shopinvader/odoo-shopinvader/blob/14.0/shopinvader_customer_price/services/customer_price.py#L24.